### PR TITLE
modified docs for 1-21

### PR DESCRIPTION
**Pull request by ahreehong**
Docs for 1-21

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
You'll probably need to rebase after this change merges: https://github.com/aws/eks-distro/pull/389

Sorry! Also I stole your changelog bullet points under "patches removed", as 1.20-2 adds these. You can remove the bullet points.

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/contents/releases/1-21/1/CHANGELOG-v1-21-eks-1.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Upgraded Kubernetes to [v1.21.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.1)
+
+### Kubernetes Components
+* etcd 3.4.15 —> 3.4.16
+* metrics-server 0.4.3 —> 0.5.0
+* pause 3.2 -> 3.5
+
+### Base Image
+
+Ungraded base image (Amazon Linux 2) version to include the latest security fixes.
```
Comment (kschumy) [inline docs/contents/releases/1-21/1/CHANGELOG-v1-21-eks-1.md:21]:
```suggestion
Upgraded base image (Amazon Linux 2) version to include the latest security fixes.
```

File: docs/contents/releases/1-21/1/release-announcement.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Amazon EKS Distro v1.21-1 is now available. This release now supports Kubernetes v1.21.1. Additional highlights of this release include the latest version of Amazon Linux 2 and upgraded versions of some Kubernetes components, such as etcd, metrics-server, and pause. Amazon EKS Distro builds of Kubernetes 1.21-1 are available through ECR Public Gallery (https://gallery.ecr.aws/?searchTerm=EKS+Distro) and GitHub (https://github.com/aws/eks-distro)
```
Comment (kschumy) [inline docs/contents/releases/1-21/1/release-announcement.txt:1]:
```suggestion
Amazon EKS Distro v1.21-1 is now available. This release now supports Kubernetes v1.21.1. Additional highlights of this release include the latest version of Amazon Linux 2 and upgraded versions of some Kubernetes components, such as etcd, metrics-server, and pause. Amazon EKS Distro 1.21-1 builds are available through ECR Public Gallery (https://gallery.ecr.aws/?searchTerm=EKS+Distro) and GitHub (https://github.com/aws/eks-distro)
```

File: docs/contents/releases/1-21/1/index.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# EKS-D v1-21-eks-1 Release
+
+For additional information, see the [changelog](CHANGELOG-v1-21-eks-1.md) for this release.
+
+## Release Manifest
+Download the release manifest here: [kubernetes-1-21-eks-1.yaml](https://distro.eks.amazonaws.com/kubernetes-1-21/kubernetes-1-21-eks-1.yaml)
```
Comment (kschumy) [inline docs/contents/releases/1-21/1/index.md:6]: We've changed how we format this doc a little. I'll explain offline what we'll need to do to update it.

File: docs/contents/index.md [hunk 4]
```diff
@@ -118,6 +118,9 @@ Machine Images), but it does use the EKS Optimized AMI. See the project
 repository for the [EKS Optimized AMI](https://github.com/awslabs/amazon-eks-ami)
 if you are interested in the AL2 container runtime kernel version.
 
+#### EKS-D 1.21 Version Dependencies
+* [v1-21-eks-1](releases/1-21/1/index.md)
```
Comment (kschumy) [inline docs/contents/index.md:122]:
```suggestion
* [v1-21-eks-1](releases/1-21/1/index.md) (July 16, 2021)
```

**Comment by kschumy** [comment]
We need to change default release branch too

**Review by kschumy — APPROVED** [review]
/lgtm

**Pull request merged**